<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popUpVisitWindow</class>
 <widget class="QDialog" name="popUpVisitWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizyta</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>29</y>
     <width>601</width>
     <height>844</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item row="11" column="1">
     <widget class="QLabel" name="visitPayment">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTextBrowser" name="visitNextVisit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Opis:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>Koszt wizyty:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextBrowser" name="visitTreatment"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Lekarz prowadzący:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="visitDoctor">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTextBrowser" name="visitDescription"/>
    </item>
    <item row="4" column="1">
     <widget class="QTextBrowser" name="visitInterview">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Następna wizyta:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Wywiad:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>Użyte materiały:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Zabieg:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QTextBrowser" name="visitMedicines"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>Użyte leki</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QTextBrowser" name="visitTools"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data wizyty:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Lekarz przyjmujący:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTextBrowser" name="visitRecommendation"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Rodzaj płatności</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="visitCost">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="visitMainDoctor">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="visitDate">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Zalecenia:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
